src/groundStation/system.py

```diff
@@ -1624,5 +1624,1071 @@ def __init__(self):
+                        'subport': 97,
+                        'inoutInfo': {
+                            'args': ['>f1', '>f1', '>f1', '>u2', '>u2', '>u1', '>u1', '>u1', '>u2', '>u1', '>u1', '>u1', '>f1', '>f1', '>f1', '>f1', '>f1', '>f1', '>f1', '>u1', '>u1', '>u1', '>?', '>?', '>u1'],
+                            'returns': {
+                                'err': '>b',
+                            }
+                        }
+                    },
+                    'ADCS_SET_CUBESENSE_CONFIG': {
+                        'subport': 98,
+                        'inoutInfo': {
+                            'args': None, #TODO: finish args, why does this have so many args ~.~
+                            'returns': {
+                                'err': '>b',
+                            }
+                        }
+                    },
+                    'ADCS_SET_MTM_CONFIG': {
+                        'subport': 99,
+                        'inoutInfo': {
+                            'args': None, #TODO: finish args
+                            'returns': {
+                                'err': '>b',
+                            }
+                        }
+                    },
+                    'ADCS_SET_DETUMBLE_CONFIG': {
+                        'subport': 100,
+                        'inoutInfo': {
+                            'args': ['>f1', '>f1', '>f1', '>f1'],
+                            'returns': {
+                                'err': '>b',
+                            }
+                        }
+                    },
+                    'ADCS_SET_YWHEEL_CONFIG': {
+                        'subport': 101,
+                        'inoutInfo': {
+                            'args': ['>f1', '>f1', '>f1', '>f1', '>f1'],
+                            'returns': {
+                                'err': '>b',
+                            }
+                        }
+                    },
+                    'ADCS_SET_TRACKING_CONFIG': {
+                        'subport': 102,
+                        'inoutInfo': {
+                            'args': ['>f1', '>f1', '>f1', '>u1'],
+                            'returns': {
+                                'err': '>b',
+                            }
+                        }
+                    },
+                    'ADCS_SET_MOI_MAT': {
+                        'subport': 103,
+                        'inoutInfo': {
+                            'args': ['>f1', '>f1', '>f1', '>f1', '>f1', '>f1'],
+                            'returns': {
+                                'err': '>b',
+                            }
+                        }
+                    },
+                    'ADCS_SET_ESTIMATION_CONFIG': {
+                        'subport': 104,
+                        'inoutInfo': {
+                            'args': ['>f1', '>f1', '>f1', '>f1', '>f1', '>f1', '>f1', '>u1', '>u1', '>u1', '>u1'],
```

Comment:
Are these '>f1' supposed to be single byte floating points? a standard float would be denoted as '>f4' meaning big endian floating point 4 bytes. a double would be '>f8'. I'm going to guess you meant '>f4'

Reply:
Ah yes thank you for pointing it out